File: skills/astra-toolkit/clients/java/examples/collections-create-collection-keyspace.java
import com.datastax.astra.client.DataAPIClient;
import com.datastax.astra.client.collections.Collection;
import com.datastax.astra.client.collections.commands.options.CreateCollectionOptions;
import com.datastax.astra.client.collections.definition.documents.Document;
import com.datastax.astra.client.databases.Database;

public class Example {

  public static void main(String[] args) {
    // Get a database
    Database database = new DataAPIClient(System.getenv("ASTRA_DB_APPLICATION_TOKEN")).getDatabase(System.getenv("ASTRA_DB_API_ENDPOINT"));

    // Create a collection
    CreateCollectionOptions options = new CreateCollectionOptions().keyspace("**KEYSPACE_NAME**");
    Collection<Document> collection =
        database.createCollection("**COLLECTION_NAME**", null, options);
  }
}

File: skills/astra-toolkit/clients/csharp/examples/collections-create-collection-keyspace.cs
using DataStax.AstraDB.DataApi;
using DataStax.AstraDB.DataApi.Core;

namespace Examples;

public class Program
{
  static async Task Main()
  {
    // Instantiate the client
    var client = new DataAPIClient();

    // Connect to a database
    var database = client.GetDatabase(
      System.Environment.GetEnvironmentVariable("ASTRA_DB_API_ENDPOINT"),
      System.Environment.GetEnvironmentVariable("ASTRA_DB_APPLICATION_TOKEN")
    );

    // Create a collection
    var collection = await database.CreateCollectionAsync(
      "**COLLECTION_NAME**",
      new CreateCollectionOptions() { Keyspace = "**KEYSPACE_NAME**" }
    );
  }
}
